src/squlearn/encoding_circuit/circuit_library/chebyshev_pqc.py

```diff
@@ -46,6 +46,8 @@ class ChebyshevPQC(EncodingCircuitBase):
                                or ``rzz`` (default: ``crz``)
         alpha (float): Maximum value of the Chebyshev Tower initial parameters, i.e. parameters
                        that appear in the arccos encoding. (default: 4.0)
+        phi_map (str): Mapping function to use for the feature encoding. Either ``arccos``
```

Comment:
I am not a fan of the name "phi_map" because it's not self-explanatory. I'd recommend something like "nonlinearity" or "nonlinear_function"